ui/src/app/views/stigmatization-page/component.spec.ts

```diff
@@ -0,0 +1,45 @@
+import {ComponentFixture, fakeAsync, TestBed, tick} from '@angular/core/testing';
+import {RouterTestingModule} from '@angular/router/testing';
+
+import {ClarityModule} from '@clr/angular';
+
+import {ServerConfigServiceStub} from 'testing/stubs/server-config-service-stub';
+
+import {
+  updateAndTick
+} from '../../../testing/test-helpers';
+
+import {ServerConfigService} from '../../services/server-config.service';
+
+import {StigmatizationPageComponent} from '../stigmatization-page/component';
+
+describe('StigmatizationPageComponent', () => {
+  let fixture: ComponentFixture<StigmatizationPageComponent>;
+  beforeEach(fakeAsync(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        RouterTestingModule,
+        ClarityModule.forRoot()
+      ],
```

Comment:
It seems like these two imports are always needed. Might be worth considering ways to have a default configuration that includes these to reduce some boilerplate.

Reply:
As it turns out, thats actually full of lies, and a lot of them don't need router testing module (although I think ClarityModule is pretty much always needed). Fixing now.